_posts/2015-07-14-mercado-pago-security-vulnerability.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Mercado Pago Security Hole"
+date: 2015-07-14 23:29:00
+categories: ["security"]
+author: etagwerker
+---
+
+On April 17th we discovered a security vulnerability in the [Mercado Pago API](https://www.mercadopago.com.ar/developers/en/api-docs/). Before disclosing this information, we contacted [Mercado Libre](http://www.mercadolibre.com/) and sent them a detailed report of the security hole and how to reproduce it.
+
+The vulnerability allowed anyone to get an access token without using the right secret for an account.
+
+You can see how the authentication process works on Mercado Pago over here: [https://www.mercadopago.com.ar/developers/en/api-docs/basics/authentication/](https://www.mercadopago.com.ar/developers/en/api-docs/basics/authentication/)
+
+## How to reproduce the bug
```

Comment:
@etagwerker I think this could give the impression that the bug is still reproducible, maybe change it to How we reproduced this bug, or something similar?